packages/app-store/routing-forms/components/RoutingNavBar.tsx

```diff
@@ -27,7 +27,7 @@ export default function RoutingNavBar({
   ];
   return (
     <div className="mb-4">
-      <HorizontalTabs tabs={tabs} />
+      <HorizontalTabs isRoutingFormsTabs tabs={tabs} />
```

Comment:
HorizontalTabs should be business logic agnostic. HorizontalTabs should not have to care if it's `isRoutingFormsTabs` or not.